numba/errors.py

```diff
@@ -94,8 +94,10 @@ def highlight(self, msg):
 
     # If the colorama version is < 0.3.9 it can break stdout/stderr in some
     # situations, as a result if this condition is met colorama is disabled and
-    # the user is warned.
-    if tuple([int(x) for x in colorama.__version__.split('.')]) < (0, 3, 9):
+    # the user is warned. Note that early versions did not have a __version__.
+    colorama_version = getattr(colorama, '__version__', '0.0.0')
+    
```

Comment:
Think this line has some extra white space that can be removed?

Reply:
Sorry about that, fixed.